code/index.html

```diff
@@ -2,21 +2,107 @@
 <html>
   <head>
     <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Project 2: The News Site</title>
     <link rel="stylesheet" href="./style.css" />
+    <link rel="icon" href="./news.png">
     <!-- dont forget the meta tag for responsiveness -->
   </head>
   <body>
     <header>
       <!-- Some kind of logo and navbar -->
+      <nav> 
+        <img class="logo" src="./news.png" alt="logo"/>
+        <h1 class="site-name">The Generic News Site</h1>
+        <div class ="navigation" id="navigation-id">
+          <ul>
+            <li><a href="#big-news-id">Big News</a></li>
+            <li><a href="#small-news-id">Small News</a></li>
+            <li><a href="mailto: contact@genericnews.no.real.mail" target="_blank">Contact</a></li>
+          </ul>
+        </div>
+      </nav>
     </header>
 
-    <section class="big-news">
+    <section class="big-news" id="big-news-id">
       <!-- The big news content -->
+      <img class="big-news-picture" src="https://picsum.photos/400/400" alt="big news"/>
+      <article class="big-news-text-block">
+        <h2 class="big-news-title">The Biggest News Right Now</h2>  
+        <p class="big-news-text">
+          I was wondering if my cat could be placed over the logo in the flyer. Can we have another option can you rework to make the pizza look more delicious we are a big name to have in your portfolio could you solutionize that for me, yet make it sexy. Can it handle a million in one go pass the mayo, appeal to the client, sue the vice president yet something summery; colourful I got your invoice...it seems really high, why did you charge so much, and the target audience is makes and famles aged zero and up, for i love it, but can you invert all colors?
+        </p>
+        <a href="#" class="read-more">Read more here!</a>
+      </article>  
     </section>
 
-    <section class="">
+    <section class="small-news" id="small-news-id">
       <!-- The other news  -->
+      <article class="small-news-card">
+        <img class="small-news-picture" src="https://picsum.photos/200/200" alt="small news">
+        <h3 class="small-news-title">X Reasons Why</h2>
+        <p class="small-news-text">
+          Three-martini lunch quick win, nor we need to harvest synergy effects but problem territories. Killing it can you send me an invite?
+        </p>
+      </article>
+      <article class="small-news-card">
+        <img class="small-news-picture" src="https://picsum.photos/200/201" alt="small news">
+        <h3 class="small-news-title">This Is What</h2>
```

Comment:
Great with naming every article, img and so on with the same class for every small news card!